### PR TITLE
bug fix, catkin clean -a is wrong command

### PR DESCRIPTION
**Pull request by yantrabuddhi**
this line gave error on my recent install, so i removed -a option

### CODE REVIEW

File: scripts/hrtool [hunk 1]
```diff
@@ -169,7 +169,7 @@ do_build_hr() {
     if [[ ! -d .catkin_tools ]]; then
         catkin init
     fi
-    catkin clean -a
+    catkin clean
```
Comment (wenwei-dev) [inline scripts/hrtool]: For compatible reason, this would be better `catkin clean -y || catkin clean -a`.

Comment (yantrabuddhi) [inline scripts/hrtool]: hi wenwei i changed it according to your suggestion

Comment (yantrabuddhi) [inline scripts/hrtool]: Hi wenwei,
 i changed it according to your suggestion

**Comment by wenwei-dev** [comment]
Fix #1

**Pull request merged**